AlfiyaSiddique/TastyTrails
# Feat: added auto-labelling PR's

## What does this PR do?

This PR added auto-labelling for all the PR's once verified by adding gssoc-ext and hacktoberfest labels

Fixes #279 

Issues: 
![image](https://github.com/user-attachments/assets/12898444-8325-4ad8-ba17-a7b3a7ceeadc)
PR's:
![image](https://github.com/user-attachments/assets/83048bd1-d7bf-4fb7-b038-8cef1b7c09cd)


## How should this be tested?


- Test 1: Created a test issue to verify if my code change effects the issue labeling, figured that it does not.
- Test 2: Raised a PR to verify if the auto-labeling works as it intends to be, it works fine.

## Checklist

### Required
